Update extensions-customscript.md with note on Update-AzureRmVM

VijayRod opened the pull request:
Update extensions-customscript.md with "Do not use Custom Script Extension to run Update-AzureRmVM with the same VM as its parameter, since it will wait on itself.".

azurecla commented:
Hi __@gangsir__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@gangsir Thank you for this contribution.  I'll have the author review.

@tamram Would you please review this change and let me know if I should merge it?  Thank you.

VijayRod committed:
Update extensions-customscript.md with note on Update-AzureRmVM

Update extensions-customscript.md with "Do not use Custom Script Extension to run Update-AzureRmVM with the same VM as its parameter, since it will wait on itself.".

mbearup reviewed (approved):
Looks good to me
